Kernel/Arch/riscv64/ThreadRegisters.h

```diff
@@ -30,6 +30,8 @@ struct ThreadRegisters {
     FlatPtr sp() const { return x[1]; }
     void set_sp(FlatPtr value) { x[1] = value; }
 
+    FlatPtr frame_pointer() const { return x[7]; }
+
```

Comment:
https://github.com/riscv-non-isa/riscv-elf-psabi-doc/blob/master/riscv-cc.adoc#frame-pointer-convention claims x8. Do we do something custom? Is the ref wrong?

Reply:
All x* indices are 1 off because we don't save x0 as that would be pretty useless.
(sp is also x2, but we use x[1] 2 lines above)